packages/core/src/odata-v4/request-builder/create-request-builder.ts

```diff
@@ -90,17 +65,6 @@ export class CreateRequestBuilderV4<EntityT extends EntityV4>
     options?: DestinationOptions
   ): Promise<EntityT> {
     this.prepare();
-    return this.build(destination, options)
-      .then(request => request.execute())
-      .then(response =>
-        deserializeEntityV4(
-          getSingleResult(response.data),
-          this._entityConstructor,
-          response.headers
-        )
-      )
-      .catch(error =>
-        Promise.reject(errorWithCause('Create request failed!', error))
-      );
+    return super.execute(destination, options);
```

Comment:
Oh no, why is `this.prepare()` still called here? Can you check whether it can be removed 🤦‍♀️

Reply:
I see what the tests say if I remove it.